component/core/src/main/java/org/exoplatform/social/core/listeners/ActivityTagMetadataListener.java

```diff
@@ -83,6 +94,11 @@ private void updateActivityTags(ExoSocialActivity activity) {
                         tagNames,
                         audienceId,
                         creatorId);
+    try {
+      listenerService.broadcast(TAG_ADDED_EVENT,activity,tagNames);
```

Comment:
the **tagNames** in this case will contain all activity tags, even the old ones ? or just the new tags added